support-frontend/assets/helpers/tracking/acquisitions.ts

```diff
@@ -139,12 +144,15 @@ function buildReferrerAcquisitionData(
 ): ReferrerAcquisitionData {
 	// This was how referrer pageview id used to be passed.
 	const referrerPageviewId =
-		acquisitionData.referrerPageviewId || getQueryParameter('REFPVID');
+		(acquisitionData.referrerPageviewId as string | undefined) ??
+		getQueryParameter('REFPVID');
+
```

Comment:
I think the real solution here is to get rid of the `Record<string, any>` type, but that feels like a real rabbit whole to fall down that I didn't want to tackle right now!